cli/js/lib.deno.ns.d.ts

```diff
@@ -1509,47 +1511,48 @@ declare namespace Deno {
    * `Uint8Array`.
    *
    * Resolves to the number of bytes written and the remote address. */
-  export function recvfrom(rid: number, p: Uint8Array): Promise<[number, Addr]>;
+  //   export function recvfrom(rid: number, p: Uint8Array): Promise<[number, Addr]>;
```

Comment:
I don’t understand why recvfrom is being removed? In any case it should be completely removed, not commented out.

Reply:
I was going to ask if it was necessary to have recvfrom exposed. But then I saw [this](https://github.com/denoland/deno/pull/4236#discussion_r387758326) and forgot to uncomment this line! Will fix that.

Reply:
Rather than leave this commented out, let's just remove it.

Reply:
Is this getting removed? I suppose if it's passing tests, it's untested - so it should be removed completely.